Formula/pocl.rb

```diff
@@ -0,0 +1,32 @@
+class Pocl < Formula
+  desc "Portable Computing Language"
+  homepage "http://portablecl.org"
+  url "http://portablecl.org/downloads/pocl-1.8.tar.gz"
+  sha256 "0f63377ae1826e16e90038fc8e7f65029be4ff6f9b059f6907174b5c0d1f8ab2"
+  license "MIT"
+  head "https://github.com/pocl/pocl.git", branch: "master"
+
+  keg_only :shadowed_by_macos
+
+  depends_on "cmake" => :build
+  depends_on "opencl-headers" => :build
+  depends_on "pkg-config" => :build
+  depends_on "hwloc"
+  depends_on "llvm"
```

Comment:
Noting that I will wait for `llvm` v14 update to see if this needs to be downgraded.